### PR TITLE
DEVDOCS-5153: [update] Store Content, blog posts default to draft status

### PR DESCRIPTION
**Pull request by slsriehl**
# [DEVDOCS-5153]

## What changed?

* Add note to indicate that blog posts default to draft status on create and update.
* clean up store_content.v2.yml

## Release notes draft

* We've updated the blog feature specification to note that the `is_published` field defaults to false on both POST and PUT. Pass `true` to avoid reverting the post to draft status. 

## Anything else?

ping {names}

[DEVDOCS-5153]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/store_content.v2.yml [hunk 1]
```diff
@@ -304,7 +304,7 @@ paths:
 
         **Notes**
 
-        * When including `published_date` in a request, supply it as a flat date string (not an object) in valid <a href="http://tools.ietf.org/html/rfc2822#section-3.3" target="_blank">RFC 2822</a>. The&#160;example request below includes a `published_date` in RFC 2822 format.
+        * When including `published_date` in a request, supply it as a flat date string (not an object) in valid <a href="http://tools.ietf.org/html/rfc2822#section-3.3" target="_blank">RFC 2822</a>. The following example request includes a `published_date` in RFC 2822 format.
```
Comment (bc-andreadao) [inline reference/store_content.v2.yml:307]:
```suggestion
        * To include `published_date` in a request, provide a flat date string (not an object) in valid <a href="http://tools.ietf.org/html/rfc2822#section-3.3" target="_blank">RFC 2822</a>. The following example request includes a `published_date` in RFC 2822 format.
```

File: reference/store_content.v2.yml [hunk 2]
```diff
@@ -1341,18 +1337,20 @@ components:
...
         summary:
           type: string
-          description: Summary of the blog post. (READ-ONLY)
+          description: Summary of the blog post. READ-ONLY.
+          readOnly: true
           example: <p>We power ecommerce websites for successful retailers all over the world</p>
         is_published:
           type: boolean
-          description: Whether the blog post is published.
+          description: Whether the blog post is published. If you want the post to be or remain published following the request, you must set this explicitly to true, even if the blog post was already published prior to the request.
```
Comment (bc-andreadao) [inline reference/store_content.v2.yml:1345]:
```suggestion
          description: Whether the blog post is published. If you want the post to be or remain published following the request, you must set the field explicitly to true, even if the blog post was already published prior to the request.
```

File: reference/store_content.v2.yml [hunk 3]
```diff
@@ -1401,18 +1400,20 @@ components:
...
```
Comment (bc-andreadao) [inline reference/store_content.v2.yml:1408]:
```suggestion
          description: Whether the blog post is published. If you want the post to be or remain published following the request, you must set the field explicitly to true, even if the blog post was already published prior to the request.
```

**Review by bc-andreadao — COMMENTED** [review]
I added a few suggestions you can take or leave

File: reference/store_content.v2.yml [hunk 4]
```diff
@@ -1271,7 +1267,8 @@ components:
             type: string
         is_published:
           type: boolean
-          description: Whether the blog post is published.
+          description: Whether the blog post is published. If you want the post to be or remain published following the request, you must set this explicitly to true, even if the blog post was already published prior to the request.
```
Comment (slsriehl) [inline reference/store_content.v2.yml:1270]:
```suggestion
          description: Whether the blog post is published. If you want the post to be or remain published following the request, you must set the field explicitly to true, even if the blog post was already published prior to the request.
```

**Comment by slsriehl** [comment]
Thanks @bc-andreadao !

**Pull request merged**